packages/dropdowns/src/elements/Menu/Items/Item.tsx

```diff
@@ -55,8 +55,12 @@ export const Item = React.forwardRef<HTMLLIElement, IItemProps>(
       }
     });
 
+    // eg of a composite component: AddItem, NextItme, PreviousItem, which is passed through the `component` prop.
+    // this is used to hide the selected SVG icon when Item is in a selected state for composite components
+    const isNotCompositeComponent = !!Component.displayName;
```

Comment:
Works, but feels a bit like magic (breaks if Garden composes a new, selectable item type in the future). Is it worth being more explicit? For example if the `Component` is specifically `AddItem` || `NextItem` || `PreviousItem`?

Reply:
I've updated this to use the exact naming of the components.